client/dashboard/sites/overview-card/style.scss

```
@@ -68,3 +68,14 @@
 .dashboard-overview-card__progress-bar {
 	width: 100% !important;
 }
+
+.dashboard-overview-card-upsell-with-action {
+	.components-card__body {
+		border-radius: 8px !important; /* stylelint-disable-line scales/radii */
```

```suggestion
@import "@wordpress/base-styles/colors";

// ...

		border-radius: $radius-large !important;
```

But I guess my question is why are we overriding core defaults at all?

You can see that `<Card>` already effectively uses 8px. There's a tweak because of an implementation detail, but it is effectively 8px already.

https://github.com/WordPress/gutenberg/blob/acdfd058ed6b027f562ff345fe8d477b15d246bc/packages/components/src/card/styles.ts#L11-L15

---

That's good context, I was wondering why it was 7px instead of 8px.

---

Why do we need to override the border-radius here 🤔

---

Removed in ce39da91b38be985610df1a1454f568fbac27fec. In dev tools I saw it was 7px instead of 8px but it's because of box-shadow 😅